generator/templates/webnn.h

```diff
@@ -105,6 +105,7 @@ typedef MLGraphBuilder (*WebnnProcCreateGraphBuilder)(MLContext context);
 typedef MLNamedInputs (*WebnnProcCreateNamedInputs)();
 typedef MLNamedOperands (*WebnnProcCreateNamedOperands)();
 typedef MLNamedOutputs (*WebnnProcCreateNamedOutputs)();
+typedef MLOperatorArray (*WebnnProcCreateOperatorArray)(MLGraphBuilder graphBuilder);
```

Comment:
The `MLGraphBuilder graphBuilder` is unnecessary like other created function.

Reply:
Replaced ```class OperatorArrayBase : public ObjectBase``` with ```class OperatorArrayBase : public RefCounted```.